figures/backfill.py

```diff
@@ -36,3 +38,15 @@ def backfill_monthly_metrics_for_site(site, overwrite=False):
         backfilled.append(dict(obj=obj, created=created, dt=dt))
 
     return backfilled
+
+
+def backfill_course_activity_date():
+    """
+    Backfill historical course_activity_date for learners who performed course activity in the past.
```

Comment:
```
Backfill historical "course_activity_date" for learners who performed course activity in the past.
```

Reply:
Done, story created for time period analysis: https://edlyio.atlassian.net/browse/EDLY-2622